ncrypt/crypt_gpgme.c

```diff
@@ -3087,10 +3087,7 @@ int pgp_gpgme_application_handler(struct Body *m, struct State *s)
       }
 
       gpgme_data_release(armored_data);
-      if (fp_out)
-      {
-        mutt_file_fclose(&fp_out);
-      }
+      mutt_file_fclose(&fp_out);
```

Comment:
Is it safe to remove the null check?

Reply:
Yes.
- `mutt_file_fclose()` checks its pointers
- This is the only code that has a pre-check before calling